docs/source/user_guide/backends/omnisci.rst

```diff
@@ -7,43 +17,42 @@ In this document it would be explained the main aspects of `ibis` and
 Modules
 -------
 
-omniscidb backend has 5 modules, which 3 of them are the main modules:
+omniscidb backend has 5 modules, the main three are:
 
 - `api`
 - `client`
 - `compiler`
 
-The `identifiers` and `operations` modules were created to organize `compiler`
-method.
+The `identifiers` and `operations` modules complement the `compiler` module.
 
 api
 ---
 
 `api` module is the central key to access the backend. Ensure to include
-the follow code into `ibi.__init__`:
+the follow code into `ibis.__init__`:
 
 .. code-block:: python
 
     with suppress(ImportError):
         # pip install ibis-framework[omniscidb]
         import ibis.omniscidb.api as omniscidb
 
-Basically, there is 3 function on `api` module>
+There are 3 functions in the `api` module:
```

Comment:
maybe you can replace `3` by `three`,  similar as you did above (https://github.com/ibis-project/ibis/pull/2265/files#diff-4ae26e229bda1f1f9f4fda583d639683R20).